content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -878,27 +918,25 @@ a known `resourceVersion` is preferable since it can achieve better performance
 of your cluster than leaving `resourceVersion` and `resourceVersionMatch` unset, which requires
 quorum read to be served.
 
+{{< table caption="resourceVersionMatch and paging parameters for list" >}}
```

Comment:
Remember to close the table shortcode: `{{< /table >}}`

Reply:
done.